connect/logger/FileLoggerWriter.hx

```diff
@@ -72,4 +72,14 @@ class FileLoggerWriter extends Base implements ILoggerWriter {
         }
         return this.file;
     }
+
+    public function setFile(): Void{
+
+    }
+
+    public function copy(): FileLoggerWriter{
+        var newCopy: FileLoggerWriter =  new FileLoggerWriter();
```

Comment:
This could be final instead of var, and type can be omitted since it is inferred by the expression.